sources/packages/api/src/route-controllers/institution-locations/institution-locations.controller.ts

```diff
@@ -48,7 +48,7 @@ export class InstitutionLocationsController extends BaseController {
   ): Promise<number> {
     // Validate the location data that will be saved to SIMS DB.
     const dryRunSubmissionResult = await this.formService.dryRunSubmission(
-      "institutionlocationcreation",
+      "institutionlocation",
```

Comment:
I would avoid changing it on this PR. The POST must include the email from the user also.